### ISSUE TITLE
GoBig and GoLatent upscaling models fail with IndexError

### ISSUE THREAD

**Issue by nikarh** [AbdBarho/stable-diffusion-webui-docker#20]
Hello, and thank you for this amazing docker image!

 Upsacling with GoBig or GoLatent fail with the fllowing exception:
```
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/gradio/routes.py", line 247, in run_predict
    output = await app.blocks.process_api(
  File "/opt/conda/lib/python3.8/site-packages/gradio/blocks.py", line 641, in process_api
    predictions, duration = await self.call_function(fn_index, processed_input)
  File "/opt/conda/lib/python3.8/site-packages/gradio/blocks.py", line 556, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/opt/conda/lib/python3.8/site-packages/anyio/to_thread.py", line 31, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/opt/conda/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 937, in run_sync_in_worker_thread
    return await future
  File "/opt/conda/lib/python3.8/site-packages/anyio/_backends/_asyncio.py", line 867, in run
    result = context.run(func, *args)
  File "scripts/webui.py", line 1731, in imgproc
    image = processGoBig(image)
  File "scripts/webui.py", line 1665, in processGoBig
    work_results.append(output_images[0])
IndexError: list index out of range

```

**Steps to Reproduce**
1. Go to 'Image Lab Tab'
2. Upload 'Single Image'
3. Select 'GoBig' upscaling model
4. Press 'Process'

**Hardware / Software:**
 - OS: Archlinux
 - GPU: RTX3080

**Additional context**
Any other context about the problem here. If applicable, add screenshots to help explain your problem.

**Comment by AbdBarho** [comment]
Bug in the main Repo https://github.com/hlky/stable-diffusion-webui/issues/464
Fix is currently in an open pull request https://github.com/hlky/stable-diffusion-webui/pull/473

I will update when it is merged.

**Comment by AbdBarho** [comment]
@nikarh the problem should be fixed in #25, can you try again from latest master?

**Comment by nikarh** [comment]
Thanks! All works now.
